observability/logging/logging-6.2/log6x-loki-6.2.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[leveloffset=+1]
+[id="log6x-loki-6-2"]
+= Storing logs with LokiStack
+:context: log6x-loki-6.2
+
+toc::[]
+
+You can configure a `LokiStack` CR to store application, audit, and infrastructure-related logs.
+
+include::snippets/log6x-loki-statement-snip.adoc[leveloffset=+1]
+
+include::modules/log6x-loki-sizing.adoc[leveloffset=+1]
+
+[id="prerequisites-6-2_{context}"]
+== Prerequisites
+
+* You have installed the {loki-op} by using the command-line interface (CLI) or web console.
+* You have a `serviceAccount` in the same namespace in which you created the `ClusterLogForwarder`.
+* The `serviceAccount` is assigned the `collect-audit-logs`, `collect-application-logs`, and `collect-infrastructure-logs` cluster roles.
+
+[id="setup-6-2_{context}"]
+== Core Set up and configuration
+
+Use role-based access controls, basic monitoring, and pod placement to deploy Loki.
+
+include::modules/log6x-loki-rbac-rules-perms.adoc[leveloffset=+1]
+include::modules/log6x-enabling-loki-alerts.adoc[leveloffset=+1]
+include::modules/log6x-loki-memberlist-ip.adoc[leveloffset=+1]
+include::modules/log6x-loki-retention.adoc[leveloffset=+1]
+include::modules/log6x-loki-pod-placement.adoc[leveloffset=+1]
+
+[id="performance-6-2_{context}"]
+== Enhanced reliability and performance
+
+Use the following configurations to ensure Loki’s reliability and efficiency in production.
+
+include::modules/log6x-identity-federation.adoc[leveloffset=+1]
+include::modules/log6x-loki-reliability-hardening.adoc[leveloffset=+1]
+include::modules/log6x-loki-restart-hardening.adoc[leveloffset=+1]
+
+[id="advanced-6-2_{context}"]
+== Advanced Deployment and Scalability
+*Specialized configurations for high availability, scalability, and error handling.*
+
+include::modules/log6x-loki-zone-aware-rep.adoc[leveloffset=+1]
+include::modules/log6x-loki-zone-fail-recovery.adoc[leveloffset=+1]
+include::modules/log6x-loki-rate-limit-errors.adoc[leveloffset=+1]
```

Comment:
- If these modules are a part of **Advanced deployment and scalability** then consider placing them one level down from the heading "Advanced deployment and scalability".

Currently the structure looks like this:

<img width="360" alt="Screenshot 2025-02-26 at 12 52 11 PM" src="https://github.com/user-attachments/assets/23a79f58-104b-4d73-a333-b414287ca6a0" />

Reply:
I'd probably change this in the future. Right now we are just trying to get the missing pieces (and there a bucket load of them!) in the 6.2 docs. The content might even be restructured soon. So keeping it the way it is for now (https://docs.openshift.com/container-platform/4.16/observability/logging/logging-6.0/log6x-loki.html). WDYT?

Reply:
Sounds fine